modules/AWS/eks/nodegroup_launch_template/templates/userdata.sh.tpl

```diff
@@ -0,0 +1,70 @@
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+
+--==MYBOUNDARY==
+Content-Type: text/x-shellscript; charset="us-ascii"
+
+### OSQUERY INSTALLATION
```

Comment:
I am thinking if we can encapsulate the os query specific logic in our code base. We don't have to expose any logic related to this as this is Atlassian specific.

Reply:
Osquery template has been moved to templates/osquery, and is processed only if the osquery fleet secret name is defined.